js/hw_3_js_react_1/src/function/Food/generateNewFood.js

```diff
@@ -0,0 +1,19 @@
+function getRandomNumber(max) {
+    return Math.floor(Math.random() * max);
+  }
+  
+  export function generateNewFood(snakeBody) {
+    let food = [[getRandomNumber(25), getRandomNumber(25)]];
+    let foodOnSnake = snakeBody.filter(
+      (item) => item[0] === food[0] && item[1] === food[1]
+    );
+  
+    while (foodOnSnake.length === 1) {
```

Comment:
This algorithm works okay when the snake is small, but the bigger it gets the longer this function will have to execute. Try to think of a better solution.